### PR TITLE
Add FBPIC

### PR DESCRIPTION
**Pull request by ax3l**
FBPIC is a Particle-In-Cell (PIC) code for relativistic plasma physics.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] ~If static libraries are linked in, the license of the static library is packaged.~
- [x] ~Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ax3l** [comment]
@conda-forge/help-python ready for review :)

**Comment by RemiLehe** [comment]
I agree to be listed in this packages as a co-maintainer.

**Comment by MKirchen** [comment]
I agree to be listed in this packages as a co-maintainer.

**Comment by soerenjalas** [comment]
I agree to be listed in this packages as a co-maintainer.

File: recipes/fbpic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set version = "0.25.0" %}
+
+package:
+  name: fbpic
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/fbpic/meta.yaml:8]:
```suggestion
```

File: recipes/fbpic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - python >=3.6
+    - setuptools
+  run:
+    - h5py
+    - numba
+    - numpy
+    - python >=3.6
+    - python-dateutil
+    - scipy
+# pyfftw
```
Comment (ocefpaf) [inline recipes/fbpic/meta.yaml:29]: Can you remove or add a comment on why this is commented out?

Comment (ax3l) [reply recipes/fbpic/meta.yaml:29]:
```suggestion
# For fast non-MKL FFTs, explore using in the future:
# pyfftw
```

File: recipes/fbpic/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/fbpic/fbpic/archive/refs/tags/{{ version }}.tar.gz
+  sha256: a868cae099fac66d20017d405caea17cb3f1aa00cb83489038f4474ca8fd28f3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (ax3l) [inline recipes/fbpic/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

**Pull request merged**